### DISCUSSION TITLE
[灌水]大家对于这个出现的Fairy有什么看法？

### DISCUSSION THREAD

**Discussion by dont-continue** [share121/inter-knot#985] [其他]
![Screenshot_20240810_195638](https://github.com/user-attachments/assets/11c4091b-ca81-4ec5-8ef1-cd9e3ce3d2c3)
众所周知，B站上出现了一个名叫fairy的人工智能号称为法厄同工作，但是自从七月以来法厄同都没有再露过面，就算去找那位<牧羊人>都声称不知道甚至到法厄同的绳网账号上找不到，但现在这个叫fairy的人工智能却跑出来了天天发视频，工作积极到超越法厄同，难不成他把法厄同给替代了？而且据说他的耗电量极大是传统的5倍甚至还有继续向上的迹象，那法厄同是怎么支付这一笔巨额费用的呢？难不成法厄同账号被盗了换了个账号赚丁尼？大家有什么看法吗？

**Comment by 1451374815** [comment]
小道消息，法厄同早被抓了，之前行动太嚣张了

**Reply by Larksonguesinaspic** [reply]
假的！！！
我前两天还看见他俩在六分街瞎逛

**Comment by gaojiaobei** [comment]
那可是传奇绳匠啊，怎么可能这么容易被抓啊。
要不消息不真，是散发出来混淆视野的，要不就是有什么内幕

**Comment by uyfh352** [comment]
一楼开玩笑，法厄同哪有那么容易被抓，说不定是法厄同抓fairy来做苦工啊

**Comment by sky27950** [comment]
小道消息，据说法厄同是被人举报的。当时被抓的时候还在喊自己不是法厄同！

**Comment by Yizakl** [comment]
为自己主人挣电费 好样的！👍👍👍

**Comment by Larksonguesinaspic** [comment]
你猜对了，他俩大号没了开了个小号

**Comment by NANTI34** [comment]
这一定是剽窃的人工智能技术的法厄同做的！他们一定是干坏事被发现换号跑路了！

**Comment by ZhuYuan-148** [comment]
最近在调查六分街录像店的超额电费一事，市民们有什么头绪吗？

**Reply by dont-continue** [reply]
我觉得很正常毕竟是录像店电视估计多一点，在当今环境下娱乐设施不多，客人多一点很正常说不定电视用的多一点

**Comment by aaa00073** [comment]
我听说啊，法厄同被人举报，已经被治安局逮捕了！

**Comment by Lanxey** [comment]
我舅舅的儿子的女朋友的堂弟的男朋友在治安局工作，他曾在某次基情后透露说：录像店的兄妹最近两个月用电量激增，但供电局调查过该店铺并没有新增大功率耗电设备，虽然还不清楚个中缘由，但治安局已经盯上了。

**Comment by MemoryZhong** [comment]
这个委托我接了（

**Comment by Yersmagit** [comment]
楼上自动接单脚本挺猖狂啊

**Comment by ijog36** [comment]
现在说不定法厄同过得挺惨呢，比如可能会因为帐号丢失出现我假扮我自己的笑话（乐）

**Comment by ei-h9** [comment]
![capture_2024-08-06_17_53_06](https://github.com/user-attachments/assets/b0fcbf5a-681f-4b7f-bf72-b4c0d235dd0c)
不到啊

**Comment by fairy-B** [comment]
疑似人机，我看过他的视频，好像真的是在给法厄同效劳，不过据说法厄同的账号没了，真是遗憾啊那可是传奇绳匠啊！！！
有个大胆的想法，法厄同有没有接到过假扮自己的委托啊QWQ

**Comment by GanlinSama** [comment]
大家都散了吧，法厄同被捕已经是板上钉钉的事了

**Comment by Inter-Knot-User** [comment]
回来吧法厄同大人
我最骄傲的信仰
历历在目的Fairy
眼泪莫名在流通

**Comment by FIREFLY26710-WEB** [comment]
**_我是六分街供电局的，小道消息，Random Play那家录像店这个月用电量比上个月翻了5倍！就说录像店哪怕全天开着电视放影片也不会有这么多吧....._**

**Comment by lllliufeng** [comment]
我倒是听说有个绳匠接单超快的，每次我看到合适的委托都秒没了:/
如果是机器队的话...

**Comment by Inter-Knot-User** [comment]
希望录像点没事☹，不然我的片子就买不到了

**Comment by su2569** [comment]
.

**Comment by xcshhhh** [comment]
逻辑：星际和平公司赞助了不少电费。
